hack/components/bump-kubevirt-ipam-controller.sh

```diff
@@ -125,6 +125,10 @@ echo 'Adjust kubevirt-ipam-controller to CNAO'
     service.beta.openshift.io/serving-cert-secret-name: kubevirt-ipam-controller-webhook-service\
 {{ end }}' Service_kubevirt-ipam-controller-webhook-service.yaml
 
+  sed -i '/        kubectl.kubernetes.io\/default-container: manager/a\{{ if .IsOpenshift }}\
```

Comment:
better imo to have please the key as something fixed like "annotations"
not "kubectl.kubernetes.io\/default-container: manager" which can change

(same for the rest when applicable)

Reply:
the deployment also has annotations. I need to put the annotation on the pod template, not on all annotations in the file.

I agree that using sed for this is wrong, we should template this properly using yaml.

Reply:
you are right
forget about non sed for now please
you did correct